src/afl-fuzz.c

```diff
@@ -2852,7 +2852,14 @@ int main(int argc, char **argv_orig, char **envp) {
 
       }
 
+      if(unlikely(afl->start_fuzzing_time == 0)){
```

Comment:
but basically this is  afl->start_time, just a tad later (and with less than a second of time difference), so I do not think this helps? just use start_time instead?

Reply:
start_time seems to set before both the initial sync() and cull_queue(), on slow harnesses (where syncing importing inputs takes a significant amount of time) the difference would be quite substantial, and that's exactly the condition I'm trying to detect with this change - the instances where the difference is substantial. I do see that this is a kinda awkward distinction though. Let me thing some more about that.

Reply:
So really, our problem could also be addressed by making a "currently_cycle" (that has value -1 or something while we are loading & syncing, 0 as soon as we enter the actual fuzzing code and then from there on is equal to cycles_done+1. 

making cycles done 0 for different states `afl->queue_cycle ? (afl->queue_cycle - 1) : 0` is kinda making it useless to detect successful startup.

Reply:
that would work too, but then you do not know for how long the the fuzzer is running. was it running for 1 second because calibration took 59:59 ?